demisto_sdk/commands/common/files/json_yml_file.py

```diff
@@ -0,0 +1,53 @@
+from abc import ABC, abstractmethod
+from functools import lru_cache
+from io import BytesIO, StringIO
+from pathlib import Path
+from typing import Any, Optional, Type, Union
+
+from pydantic import Field, validator
+
+from demisto_sdk.commands.common.files.text_file import TextFile
+from demisto_sdk.commands.common.handlers.xsoar_handler import XSOAR_Handler
+
+
+class JsonYmlFile(TextFile, ABC):
```

Comment:
thats not a great name. Maybe "StructuredDataFile" or something that better indicates what this is meant to be

Reply:
this is why i called it `HandlerFile` before because its a file that should be used by `xsoar_handler` 
In the graph for example we have classes called IntegrationScript

so why do you think J`sonYmlFile` isn't good enough, do you have maybe other suggestion?